arcade/pymunk_physics_engine.py

```diff
@@ -82,6 +83,7 @@ def add_sprite(self,
             :param max_vertical_velocity: maximum velocity on the y axis
             :param radius:
             :param collision_type:
+            :param moment_of_intertia: Deprecated alias of moment_of_inertia compatible with a typo introduced in 2.6.2
```

Comment:
This has the extra t in it.

Reply:
That version of the mispelled argument was added at the end of the arguments intentionally to preserve compatibility with 2.6.2. That's why the docstring says deprecated after the mispelled version.

There's a correctly spelled version of the keyword earlier in the arguments list.